lpvartypes.pas

```diff
@@ -1776,6 +1777,20 @@ function TLapeType.Eval(Op: EOperator; var Dest: TResVar; Left, Right: TResVar;
     end;
   end;
 
+  function TryOperatorOverload(): TResVar;
+  begin
+    Result := NullResVar;
+    if (op in OverloadableOperators) and Right.HasType() and Left.HasType() and (not(op in UnaryOperators)) then
```

Comment:
Is `(not(op in UnaryOperators))` required here? Because `(op in OverloadableOperators)` should take care of that.